source/core/src/main/com/csse3200/game/entities/Entity.java

```diff
@@ -460,18 +460,42 @@ public void read(Json json, JsonValue jsonMap) {
                     TerrainTile terrainTile = ServiceLocator.getGameArea().getMap().getTile(tile.getPosition());
                     terrainTile.setOccupant(tile);
                     break;
+	            case ShipPartTile:
```

Comment:
is the indenting here correct?

Reply:
yeah idk what's up with that, has the same indenting as everything else when I open the file. 
will see if it's fixed when I push the debris stuff

Reply:
> yes it is, you can hit 'expand up' to see more of the context btw

still shows it to be inline with the break so would never reach, at least for me

Reply:
was a whitespace issue, fixed